bibtex-actions.el

```diff
@@ -355,65 +361,68 @@ personal names of the form 'family, given'."
-             (parsebib-parse files :fields (bibtex-actions--fields-to-parse))
-             using (hash-key citekey)
-             collect
-             (let* ((files
-                     (when (or (bibtex-actions-get-value
-                                bibtex-actions-file-variable candidate)
-                               (bibtex-actions-file--files-for-key
-                                citekey bibtex-actions-library-paths bibtex-actions-file-extensions))
-                       " has:files"))
-                    (notes
-                     (when (bibtex-actions-file--files-for-key
-                            citekey bibtex-actions-notes-paths bibtex-actions-file-extensions)
-                       " has:notes"))
-                    (link (when (bibtex-actions-has-a-value '("doi" "url") candidate)
-                            "has:link"))
-                    (candidate-main
-                     (bibtex-actions--format-entry
-                      candidate
-                      star-width
-                      (car bibtex-actions-template)))
-                    (candidate-suffix
-                     (bibtex-actions--format-entry
-                      candidate
-                      star-width
-                      (cdr bibtex-actions-template)))
-                    ;; We display this content already using symbols; here we add back
-                    ;; text to allow it to be searched, and citekey to ensure uniqueness
-                    ;; of the candidate.
-                    (candidate-hidden (s-join " " (list files notes link context citekey))))
-               (cons
-                ;; If we don't trim the trailing whitespace,
-                ;; 'completing-read-multiple' will get confused when there are
-                ;; multiple selected candidates.
-                (s-trim-right
-                 (concat
-                  ;; We need all of these searchable:
-                  ;;   1. the 'candidate-main' variable to be displayed
-                  ;;   2. the 'candidate-suffix' variable to be displayed with a different face
-                  ;;   3. the 'candidate-hidden' variable to be hidden
-                  (propertize candidate-main 'face 'bibtex-actions-highlight) " "
-                  (propertize candidate-suffix 'face 'bibtex-actions) " "
-                  (propertize candidate-hidden 'invisible t)))
-                (cons citekey candidate))))))
-
-(defun bibtex-actions--affixation (cands)
-  "Add affixation prefix to CANDS."
-  (cl-loop
-   for candidate in cands
-   collect
-   (let ((candidate-symbols (bibtex-actions--symbols-string
-                             (string-match "has:files" candidate)
-                             (string-match "has:note" candidate)
-                             (string-match "has:link" candidate))))
-     (list candidate candidate-symbols ""))))
+    (seq-map
+     (lambda (entry)
+       (let* ((citekey (bibtex-actions-get-value "=key=" entry))
```

Comment:
Will this work for csl json?

I wasn't clear how to get the key from the hash here.